com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/configuration/SpringConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package com.sap.cloud.lm.sl.cf.web.configuration;
+
+import java.net.URL;
+import java.util.Map;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.InternalAuthenticationServiceException;
+import org.springframework.web.client.RestTemplate;
+
+import com.sap.cloud.lm.sl.cf.client.uaa.UAAClient;
+import com.sap.cloud.lm.sl.cf.client.uaa.UAAClientFactory;
+import com.sap.cloud.lm.sl.common.util.JsonUtil;
+
+@Configuration
+public class SpringConfiguration {
```

Comment:
On second thought, UAAClientConfiguration might be a better name for this class.